app/controllers/contributors_controller.rb

```diff
@@ -56,6 +61,11 @@ def count
     render json: { count: Contributor.with_tags(params[:tag_list]).count }
   end
 
+  def conversations
+    @messages = @contributor.conversations.includes(%i[files photos request recipient sender])
+    @reply_to = @contributor.replies.find(params[:reply_to]) if params[:reply_to].present?
```

Comment:
I usually try to always use strong params, but since you are never calling params without specfically referencing which param, I think it doesn't leave room for vulnerabilities.